src/client/src/containers/ReviewAndGenerate/index.tsx

```diff
@@ -31,19 +35,27 @@ const messages = defineMessages({
     id: "context.almostDone",
     defaultMessage:
       "You're almost done - review your project details on the right and make any necessary adjustments!"
+  },
+  quickStart: {
+    id: "context.quickStart",
+    defaultMessage: `Woo-hoo! You skipped to the end where we'll get you started quickly. Just click ""Create Project".`
```

Comment:
![image](https://user-images.githubusercontent.com/32103099/62253531-2f4c4400-b3c4-11e9-9200-0e190270845a.png)

Reply:
@njindal239  You seem to have an extra " here

Reply:
Whoops! Thanks